src/ontology/uberon-edit.obo

```diff
@@ -214663,10 +214663,13 @@ property_value: provenance_notes "This class was sourced from an external ontolo
 
 [Term]
 id: UBERON:3011045
-name: gasserian ganglion
+name: obsolete gasserian ganglion
 synonym: "gasserian ganglia" RELATED [UBERON:3011045]
```

Comment:
move the synonym to the other term (the one that is the replacement for this one)

Reply:
ok if you say so. Definitely out of my depth. Will do

Reply:
Ah. It is already there as

synonym: "Gasserian ganglia" RELATED PLURAL [Wikipedia:Trigeminal_ganglion]

I have checked for it but I must have missed the case insensitive flag.

So nothing to do here.

Reply:
No worries, just delete it from here then!